simple_history/manager.py

```diff
@@ -66,16 +167,37 @@ def most_recent(self):
         return self.instance.__class__(**values)
 
     def as_of(self, date):
-        """Get a snapshot as of a specific date.
+        """
+        Get a snapshot as of a specific date.
+
+        The resulting queryset will locate the most recent historical record
+        before the specified date for each primary key, generating instances.
+        If the most recent historical record is a deletion, that instance is
+        dropped from the result.
```

Comment:
I don't understand what this actually means. Can you clarify here and maybe together we can come up with alternative language?

Reply:
If you are querying for instances (using the as_of query), and the latest historical record (based on your timepoint in the query) is a deletion record, that instance is not included in the result.  This behavior is identical to what was there before.

If you are querying for historical records, and the latest historical record (based on your timepoint in the query) is a deletion record, you will get it.

Reply:
Alright, thank you for the explanation.